src/pd/Client.cc

```diff
@@ -202,6 +202,7 @@ void Client::switchLeader(const ::google::protobuf::RepeatedPtrField<std::string
         return;
     }
 
+    log->debug("switch leader from " + old_leader + " to " + leader);
```

Comment:
log it as info level? info level is enabled by default and I think switching leader is not frequent.

Reply:
done